### ISSUE TITLE
unused param

### ISSUE THREAD

**Issue by mybigman** [nlepage/go-cuid2#2]
length is not used here?

```go
func hash(input string, length int) (string, error) {
	return bufToBigInt(sha3.Sum512([]byte(input))).Text(36)[1:], nil
}
```

**Comment by nlepage** [comment]
Yes this parameter was originally copied from the JS version, and it has been since removed from the JS version, see https://github.com/paralleldrive/cuid2/commit/54ef5f9db5168e52b4b0edb125d003317ff15dea#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556
Overall, this Go version would need to be updated to match the latest JS version, and also documented.
